docs/source/conf.py

```diff
@@ -33,6 +33,7 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
+nbsphinx_allow_errors = True
```

Comment:
I suggest not including this flag for all notebooks. This would fail silently and lead to some errors being shown on the docs page.